src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const element = document.getElementsByClassName('population');
+let allPopulationNumber = 0;
+
+for (const key of element) {
+  allPopulationNumber += +key.innerHTML.replaceAll(',', '');
+}
+
+const averagePopulation = allPopulationNumber / element.length;
```

Comment:
The calculated averagePopulation should be formatted with a thousands separator to match the numeric style of the populations presented, as required by the task description.